docs/dev/searxng_extra/update.rst

```diff
@@ -0,0 +1,68 @@
+========================
+``searxng_extra/update``
+========================
+
+Scripts to update static data in :origin:`searx/data/`
+
+.. _update_ahmia_blacklist.py:
+
+``searxng_extra/update/update_ahmia_blacklist.py``
```

Comment:
These subheadings all start with `searxng_extra/update/` ... I think we can omit that since it's already the page title.

Reply:
>  I think we can omit that since it's already the page title.

That is true, fix is on the way ...

> It would be nice if the script source code was linked (maybe via Sphinx?).

What do you men by? .. except on the module itself, all function etc have a [source] link .. 

![grafik](https://user-images.githubusercontent.com/554536/148228564-72cf4ec8-73c2-4bf4-9eb3-3d0fd79b4783.png)

Reply:
Yeah I mean the module itself because several of those modules apparently don't have any public API members.

Reply:
> Yeah I mean the module itself ...

I don't know how / AFAIK there is no option; we use https://www.sphinx-doc.org/en/master/usage/extensions/viewcode.html#module-sphinx.ext.viewcode

Reply:
A added them [source] link manually https://return42.github.io/searxng/dev/searxng_extra/update.html#update-ahmia-blacklist-py

![grafik](https://user-images.githubusercontent.com/554536/148241896-084ed467-db45-4dba-bfca-fac18a458274.png)

Reply:
Thanks!